manuscript/algorithm.tex

```diff
@@ -182,3 +191,69 @@ \subsubsection{\gls{MARKAL}}
 of all the coefficients for the \gls{LP} problem, and is
 solved. 
 
+
+\subsection{Stochastic-Optimizing (SO)}
+Stochastic optimization refers to optimization methods
+that incorporate randomness in either the objective function
+or the constraints \cite{lauren_stochastic_2015}. It aims to find
+zeros for the objective function by generating random variables.
+This makes it fit for problems that contain uncertainty, which
+real world problems mostly are. 
+The random variables are defined by a probability distribution
+or sometimes just a few values.
+
+Mathematically, the method attempts to find properties of $f$ without
+evaluating $f$ directly, but using random samples of $F(\theta, \xi).$
+
+\begin{align}
+	f(\theta)= E_{\xi} [F(\theta, \xi)] \\
+	\xi = \mbox{random variable} \\
+	f(\theta) = \mbox{objective function} \\
+	F(\theta,\xi) = \mbox{ random samples created} \\
+\end{align}
+
+Few examples of the Stochastic-Optimizing method are
+the Markov Switching Model and the Gaussian Process Regression.
+
+
+\subsubsection{Markov Switching Model}
+The Markov Switching Model depends on the idea that
+the future is independent of the past, but only dependent
+on the present. It utilizes Markov Chains, which 
+are mathematical structures that either stay or hop to another
+'state', with certain probabilities.
```

Comment:
This explanation of markov chains could be made more intuitive. Maybe something like "Markov Chains, which characterize the probability of a system transitioning between states."